src/core_plugins/metrics/public/components/panel_config/table.js

```diff
@@ -0,0 +1,155 @@
+import React, { Component, PropTypes } from 'react';
+import FieldSelect from '../aggs/field_select';
+import SeriesEditor from '../series_editor';
+import { IndexPattern } from '../index_pattern';
+import createTextHandler from '../lib/create_text_handler';
+import createSelectHandler from '../lib/create_select_handler';
+import uuid from 'uuid';
+import YesNo from '../yes_no';
+
+class TablePanelConfig extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { selectedTab: 'data' };
+  }
+
+  componentWillMount() {
+    const { model } = this.props;
+    const parts = {};
+    if (!model.bar_color_rules || (model.bar_color_rules && model.bar_color_rules.length === 0)) {
+      parts.bar_color_rules = [{ id: uuid.v1() }];
+    }
+    this.props.onChange(parts);
+  }
+
+  switchTab(selectedTab) {
+    this.setState({ selectedTab });
+  }
+
+  render() {
+    const { selectedTab } = this.state;
+    const defaults = { drilldown_url: '', filter: '', pivot_label: '', pivot_rows: 10 };
+    const model = { ...defaults, ...this.props.model };
+    const handleSelectChange = createSelectHandler(this.props.onChange);
+    const handleTextChange = createTextHandler(this.props.onChange);
+    let view;
+    if (selectedTab === 'data') {
+      view = (
+        <div>
+          <div className="vis_editor__table-pivot-fields">
+            <div className="vis_editor__container">
+              <div className="vis_ediotr__vis_config-row">
+                <p>
+                  For the table visualization you need to define a field to
+                  group by using a terms aggregation.
+                </p>
+              </div>
+              <div className="vis_editor__vis_config-row">
+                <label className="vis_editor__label">Group By Field</label>
```

Comment:
I've just merged the a11y TSVB PR. Could you maybe rebase to master and also label this, the `FieldSelect` now supports an `id` property, and there is the `htmlIdGenerator` to generate the ids. (I skip a long explanation here, since you reviews that PR anyway :D)